CONTRIBUTING.rst

```diff
@@ -526,6 +526,41 @@ checkout of MLflow:
 These commands generate a new migration script (e.g., at ``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``)
 that you should then edit to add migration logic.
 
+Docker development (experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Instead of setting up local or virtual environment, it's possible to write code and tests inside a Docker container that will contain an isolated Python environment setup inside.
+It's possible to build and run preconfigured image, then attach with the compatible code editor (e.g. VSCode) into running container. This helps avoiding issues with local setup, e.g. on CPU architectures that are not yet fully compatible with all dependency packages (e.g. Apple M-architecture).
+
+Prerequisites
++++++++++++++
+- Docker runtime installed on a local machine (https://docs.docker.com/get-docker/)
+- Code editor compatible capable of running inside containers
+  - Example: VSCode (https://code.visualstudio.com/download) with Remote Containers extension (https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+Setup
++++++
+
+Run the following command:
+
+.. code-block:: bash
+
+    dev/run-test-container.sh
+
+You will need to wait until the docker daemon will complete building the docker image. After successful build, the container will be automatically run with ``mlflow-test`` name. A new shell session running in container's context will start in the terminal window, do not close it.
+
+Now you can attach to the running container with your code editor.
+
+Instructions for VSCode: 
+  - invoke the command palette (``[Ctrl/CMD]+Shift+P``)
+  - find "Remote-Containers: Attach to Running Container..." option, confirm with ``Enter`` key
+  - find the "mlflow-test" container, confirm with ``Enter`` key
+  - a new code editor should appear running inside the context of Docker container
+  - you can now freely change source code and corresponding tests, the changes will be reflected on your machine filesystem
+  - to run code or tests inside container, you can open a terminal with ``[Ctrl/CMD]+Shift+``` and run any command which will be executed inside container, e.g. ``pytest tests/test_import.py``
```

Comment:
```suggestion
  - to run code or tests inside container, you can open a terminal with ``[Ctrl/CMD]+Shift+``` and run any command which will be executed inside container, e.g. ``pytest tests/test_version.py``
```


`test_import.py` might not work because it requires docker. I think `test_version.py` should work.

Reply:
Fixed, thanks!